spot-client/src/common/vendor/jitsi-meet-js-provider.js

```diff
@@ -9,7 +11,12 @@ export default {
             return this._jitsiMeetJS;
         }
 
-        JitsiMeetJS.init({});
+        try {
+            JitsiMeetJS.init({});
+        } catch (error) {
+            logger.error('Failed to initialize JitsiMeetJS', { error });
```

Comment:
I'm thinking it should be left undefined after it fails to initialize. Or is it still used later on ?

Reply:
If it's used later on then it's a bit weird, because right now we probably assume that we know what went wrong, but that might not be the case in the future and may be causing unexpected problems.

Reply:
Spot-TV needs the init for gUM related actions. Spot-Remote does not. Both need it for XMPP connection. Maybe the better thing would be to no longer call get() and instead wrap all JitsiMeetJS calls to avoid exposing the global entirely. What do you think?

Reply:
yes, maybe that will be better

Reply:
also I was thinking about extract xmpp related stuff to another project, but that's more work